packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql

```diff
@@ -1,16 +1,18 @@
 type Blog @model {
   id: ID!
   name: String!
-  posts: [Post] @connection(name: "BlogPosts")
+  posts: [Post] @connection(keyName: "byBlog", fields: ["id"])
 }
-type Post @model {
+
+type Post @model @key(name: "byBlog", fields: ["blogID"]) {
   id: ID!
   title: String!
-  blog: Blog @connection(name: "BlogPosts")
-  comments: [Comment] @connection(name: "PostComments")
+  blogID: String!
+  comments: [Comment] @connection(keyName: "byPost", fields: ["id"])
```

Comment:
We should also add the "belongs to" connection into the Post model definition just to have the same behavior as the previous setup.
blog: Blog @connection(fields: ["blogID"])

Reply:
Changed